app/controllers/api/v1/planes_controller.rb

```diff
@@ -13,4 +29,13 @@ def latest_planes(offset = 0, limit = nil)
   def plane_index_params
     params.permit(:offset, :limit)
   end
+
+  def admin?
+    @current_user.id == 1
+  end
```

Comment:
admin code should happen inside the User model. Cause admin checking will not happen only in Plane controller.